doc.config.json

```diff
@@ -366,11 +366,11 @@
         "support": [
           {
             "title": "Support",
-            "path": "/docs/support/_index.mdx",
+            "path": "/docs/support/support.md",
```

Comment:
This should still be _index otherwise the url path will be support/support

Reply:
@davemooreuws this should be fixed now